codeworld-compiler/test/testcase/test1/saved_error.hs

```diff
@@ -0,0 +1,2 @@
+program.hs:1:1: error:
```

Comment:
Yes, actually I was completing the part and all of a sudden my system started creating some problem, and I just pushed the code in between.
I'll just update it.